sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/KafkaReadSchemaTransformProviderTest.java

```diff
@@ -120,7 +120,6 @@ public void testBuildTransformWithAvroSchema() {
         KafkaReadSchemaTransformConfiguration.builder()
             .setTopic("anytopic")
             .setBootstrapServers("anybootstrap")
-            .setFormat("avro")
             .setSchema(AVRO_SCHEMA)
```

Comment:
is .setSchema sufficient [ if AVRO_SCHEMA ], that we don't need AVRO in setFormat?  [ I probably need to read this class to see if this gets auto set ]

Reply:
Or, is the expectation that this test will now fail without setFormat?

Reply:
The original test didn't have the setFormat so I let the test like it was before to ensure compatibility

Reply:
I don't fully understand this ...  since the diff of the PR shows removing the line.  BUT, since tests aren't failing, I guess this isn't causing a problem.  Would like to add even more test cases, BUT, I don't think required or needed to hold things up.